Formula/ruby.rb

```diff
@@ -36,8 +35,8 @@ class Ruby < Formula
   # The exception is Rubygem security fixes, which mandate updating this
   # formula & the versioned equivalents and bumping the revisions.
   resource "rubygems" do
-    url "https://rubygems.org/rubygems/rubygems-2.6.14.tgz"
-    sha256 "406a45d258707f52241843e9c7902bbdcf00e7edc3e88cdb79c46659b47851ec"
+    url "https://rubygems.org/rubygems/rubygems-2.7.3.tgz"
```

Comment:
@DomT4 thoughts on bumping this? My inclination is to leave it alone at 2.6.14

Reply:
There are some pretty nice changes in the new versions of `gem`, so if it works across all our versioned formulae I'd be tempted to bump, but sticking my maintainer hat back on for a moment I'd certainly agree there's nothing that _needs_ to be pushed out to users. `gem update --system` is quite a well-known command, available to anyone who cares particularly. I lean towards updating, but it's only a gentle lean.

Reply:
@lembacon OK let's try updating gem in ${all_the_rubies} and revision bumping them, revision bumping all the rev deps of ruby since things seem to have decided to link based on the patch-level version of the library :see_no_evil:

Reply:
Some of the other versioned formulae (`ruby@2.2` and `ruby@2.3`) have new releases (I think?) so they can be updated rather than revision bumped at least. Less hideousness 🙈.